dubbo-config/dubbo-config-api/src/main/java/org/apache/dubbo/config/AbstractReferenceConfig.java

```diff
@@ -102,7 +102,7 @@ public void setInit(Boolean init) {
         this.init = init;
     }
 
-    @Parameter(excluded = true)
+    @Parameter(excluded = true, excludedMetadata = true)
     public Boolean isGeneric() {

```

Comment:
make the function isGeneric excludedMetadata, it will fix the bug( in centos, if getMethods 
 the function isGeneric after getGeneric it will be wrong make the default null to false)